pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -884,13 +860,21 @@ func GetBootstrapClusterRoles() []rbac.ClusterRole {
 	finalClusterRoles := []rbac.ClusterRole{}
 	finalClusterRoles = append(finalClusterRoles, openshiftClusterRoles...)
 	finalClusterRoles = append(finalClusterRoles, openshiftControllerRoles...)
+
+	// add default metadata only to openshift cluster roles
+	for i := range finalClusterRoles {
+		role := &finalClusterRoles[i]
+		addDefaultMetadata(role)
```

Comment:
Seems like this should be done when you're building the roles.  What is the use-case for having `GetOpenshiftBootstrapClusterRoles` to be missing annotations?